### PR TITLE
Fix style in README example

### PR DESCRIPTION
**Pull request by ghoneycutt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,12 +77,14 @@ Or using a plain text file stored on the filesystem:
...
--> ipset { 'bar':
-  ensure => present,
-  set    => 'file:///tmp/bar_set_content',
-  type   => 'hash:net',
+
+ipset { 'bar':
+  ensure    => present,
+  set       => 'file:///tmp/bar_set_content',
+  type      => 'hash:net',
+  subscribe => File['/tmp/bar_set_content'],
```
Comment (ekohl) [inline README.md]: This changes it from a `require` to `subscribe`. Is that intentional?

Comment (bastelfreak) [reply README.md]: Personally, I prefer `->` over `subscribe/require`.

Comment (alexjfisher) [reply README.md]: Style guide (which we should either follow or submit PRs against) prefers the metaparameters. https://puppet.com/docs/puppet/latest/style_guide.html#chaining-arrow-syntax

Comment (ghoneycutt) [reply README.md]: Just changing arrows to words, which is the preferred way to handle relationships.

Comment (ekohl) [reply README.md]: `->` would translate to `require`. `~>` would be `subscribe`.

Comment (Dan33l) [reply README.md]: https://puppet.com/docs/puppet/latest/lang_relationships.html#refreshing-and-notification

Comment (ghoneycutt) [reply README.md]: I get it was a require, but it should be a notify. If the networks change, then the ipset should change, too.

**Comment by ghoneycutt** [comment]
bump

**Pull request merged**